src/site/includes/images_list_with_additional_info_component.drupal.liquid

```diff
@@ -0,0 +1,26 @@
+<!-- this template needs to be included inside a <dl/> -->
+<!-- media list -->
+{%if images.size != 0%} 
+    {% assign pageSize = 10 %}
+    {% assign remainingImages = images | sliceArrayFromStart: pageSize, images.size %}
+        {% for mediaImage in images %}
+            {% assign recordsCount = recordsCount | plus: 1 %}
+            {% unless recordsCount > pageSize %}
+                {% include "src/site/includes/image_detail_mobile_responsive.drupal.liquid" 
+                with mediaImage = mediaImage  %}
+            {% endunless %}
+        {% endfor %}
+    {% if recordsCount > pageSize %}
+        <div class="form-expanding-group additional-info-container vads-u-margin-y--2p5">
+            <span class="additional-info-title">Show more</span>
+            <span>
+                <div class="additional-info-content">
+                {% for remainingImage in remainingImages %}
+                    {% include "src/site/includes/image_detail_mobile_responsive.drupal.liquid" 
```

Comment:
Leaving this here for myself/other reviews

Link to `src/site/includes/image_detail_mobile_responsive.drupal.liquid`: https://github.com/department-of-veterans-affairs/vets-website/pull/15270/files#diff-45a75abadb6bef1c40c2e4202e5f5e9d4155156b97c3563185bb994a055463dfR1